charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -284,7 +280,7 @@ spec:
       {{- end }}
       initContainers:
       - command:
-        - cp
+        - /bin/cp
```

Comment:
Here is still `command` .
https://github.com/argoproj/argo-cd/blob/v2.7.0/manifests/base/repo-server/argocd-repo-server-deployment.yaml#L198-L201

<img width="559" alt="image" src="https://user-images.githubusercontent.com/30188755/235654725-01e6a44c-b43e-4589-860c-f7175d9d0d2c.png">